src/agni/client/receiver/ChatReceiver.java

```diff
@@ -1,22 +1,37 @@
 package agni.client.receiver;
 
-import agni.client.view.AgniClientView;
+import java.io.UnsupportedEncodingException;
+import java.util.Arrays;
 
 public class ChatReceiver extends MessageParser {
-
+	
     public ChatReceiver() {
-
+    	
     }
 
     private void notifyChatReceived(String src, String message) {
```

Comment:
The argument `src` is not very descriptive, probably a better name would be good.